pkg/alertmanager/alertmanager.go

```diff
@@ -340,6 +343,43 @@ func (am *Alertmanager) GetReceiversHandler(w http.ResponseWriter, _ *http.Reque
 	}
 }
 
+func (am *Alertmanager) TestReceiversHandler(w http.ResponseWriter, r *http.Request) {
+	payload, err := io.ReadAll(r.Body)
+	if err != nil {
+		http.Error(w,
+			fmt.Sprintf("error reading request body: %s", err.Error()),
+			http.StatusBadRequest)
+	}
+
+	c := alertingNotify.TestReceiversConfigBodyParams{}
+	err = json.Unmarshal(payload, &c)
+	if err != nil {
```

Comment:
Shorter, does the same thing:
```suggestion
	var c alertingNotify.TestReceiversConfigBodyParams
	if err := json.NewDecoder(r.Body).Decode(&c); err != nil {
```